webapp/app/containers/Stats/StatisticFilter/reducers.js

```diff
@@ -6,7 +6,7 @@ import {
   GET_FILTERS,
```

Comment:
Maybe try to write some more accurate type name, since redux go through every reducer once an action is fired, we might triger some conflicts if not